packages/components/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import {
+  DisclosureProps,
+  Menu as AriakitMenu,
+  MenuButton,
+  MenuItem,
+  useMenuState,
+} from "ariakit";
+import map from "lodash/map";
+import { Button } from "../Button";
+import { Box } from "../../primitives/Box";
+
+export type MenuItemProps = {
+  label: string;
+  onClick: () => void;
+};
+
+export type MenuProps = {
+  /** The button element to open the menu and display the list of menu items. */
+  menuButton?: JSX.Element;
+  /** The list of menu items. */
+  items: MenuItemProps[];
+};
+
+const EllipsisButton = (
+  <Button
+    aria-label="Open menu"
+    iconOnly
+    leadingIcon="EllipsisVerticalIcon"
+    type="button"
+    variant="ghost"
+  />
+);
+
+/** A menu is a button element that opens a menu with items. */
+const Menu = React.forwardRef<HTMLButtonElement, MenuProps>(
+  ({ menuButton, items }, ref) => {
+    const state = useMenuState();
+
+    const button = menuButton || EllipsisButton;
+
+    return (
+      <div>
+        <MenuButton ref={ref} state={state} {...button.props}>
+          {(buttonProps: DisclosureProps) =>
+            React.cloneElement(button, buttonProps)
+          }
+        </MenuButton>
+
+        <AriakitMenu state={state}>
+          <Box.div
+            borderColor="colorBorderPrimary"
+            borderRadius="borderRadius20"
+            borderStyle="borderSolid"
```

Comment:
```suggestion
            borderStyle="borderStyleSolid"
```

Reply:
I tried that before, but it's an invalid value:
![image](https://github.com/Localitos/pluto/assets/6059188/7c007d53-4a96-4eff-9bab-7702dcf2046b)

Reply:
That is strange because for me I have the opposite problem. Maybe @rssilva or @apvale could see what they get?

Reply:
`yarn build` resolved the issue.